src/sqlContextUtils.js

```diff
@@ -260,7 +260,15 @@ function addState( fsm, name, stepAction ) {
 		_onEnter: function() {
 			var promise;
 			var exec = function( options ) {
-				promise = executeSql( fsm, name, options );
+				// Capture call stack for call to execute from within a step.
+				// This provides call context that currently gets lost when we
```

Comment:
Probably ought to write this in the past tense, for future readers: "This provides call context that previously got lost..."